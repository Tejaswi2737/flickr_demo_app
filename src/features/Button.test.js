import React from 'react';
import renderer from 'react-test-renderer';
import {
    DefaultButton,
} from './Button.stories';

it('renders correctly', () => {
    const tree = renderer.create(<DefaultButton {...DefaultButton.args} />).toJSON();
    expect(tree).toMatchSnapshot();
});
