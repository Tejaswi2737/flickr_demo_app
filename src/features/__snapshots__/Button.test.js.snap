// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-gsDKAQ efWQeV"
  onClick={[Function]}
>
  <div
    className="sc-bdvvtL kNTctQ"
    color="darker"
    height="medium"
    size="medium"
  >
    Button
  </div>
</div>
`;
